python/ray/serve/schema.py

```diff
@@ -176,6 +176,11 @@ class DeploymentSchema(
         default=None, description="Options set for each replica actor."
     )
 
+    is_driver_deployment: bool = Field(
+        default=False,
+        description="Indicate Whether the deployment is driver deployment",
```

Comment:
and what does the driver deployment mean here? 
```suggestion
        description="Indicate Whether the deployment is driver deployment. Driver deployments are spawned one per node.",
```